src/js/components/NameValueList/index.d.ts

```diff
@@ -3,6 +3,7 @@ import { AlignType, WidthType } from '../../utils';
 import { GridProps } from '../Grid';
 export interface NameValueListProps {
   align?: AlignType;
+  children?: React.ReactNode;
```

Comment:
more of a question in some of the files you added the ` children?: React.ReactNode;`
as well as adding `children` to the omit` Omit<ButtonType, 'icon' | 'size' | 'children'> {}`  just trying to understand why this is included in certain files?

Reply:
I am omitting the `children` prop and then redefining it in certain cases if the `children` prop that was given through the intrinsic element or interface has a different type definition from what we need.

Reply:
thank you for clarifying